src/library-authoring/author-library/specs/LibraryAuthoringPage.spec.jsx

```diff
@@ -323,4 +326,19 @@ testSuite('<LibraryAuthoringPageContainer />', () => {
       () => expect(deleteLibraryBlock.fn).toHaveBeenCalledWith({ blockId: block.id }),
     );
   });
+
+  it('Rename library', async () => {
+    const library = libraryFactory();
+    const block = blockFactory(undefined, { library });
+    await render(library, genState(library, [block]));
+    
+    const editButton = screen.getByRole('button', { name: /edit name button/i })
```

Comment:
nit: its working, but why do you use lower case `/edit name button/I`? alt="Edit name button" is written with capitalise letter.
Its little bit confusing.

also below